code/datums/supplypacks.dm

```diff
@@ -309,6 +309,15 @@ var/list/all_supply_groups = list("Supplies","Clothing","Security","Hospitality"
 	containername = "silicate tank crate"
 	group = "Supplies"
 
+/datum/supply_packs/sacidtank
+	name = "Sulphuric acid tank"
+	contains = list(/obj/structure/reagent_dispensers/sacid)
+	cost = 8
```

Comment:
That's a lot for 8 credits. I think the others were balanced around their commonness in price.

That said it's not a dealbreaker.

Reply:
Yeah, I wasn't 100% on the price. I'll raise it later if we start seeing too much of these things.